index.html

```diff
@@ -334,18 +341,11 @@ <h2 class="javaS">CSS</h2>
       </div>
     </div>
       <footer>
-        <div class="final-section">
+        <!-- here -->
+        <div class="final-section" id="the-final"> 
           <div class="contact-me-img">
-            <img
-              class="contactme-img"
-              src="Microverse-Student-Project-1/Contact.png"
-              alt="background-img"
-            />
-            <img
-              class="contact-shapes"
-              src="Microverse-Student-Project-1/Contact2.png"
-              alt="background-img"
-            />
+            <img class="contactme-img"  src="Microverse-Student-Project-1/Contact.png"   alt="background-img"/>
+            <img class="contact-shapes"  src="Microverse-Student-Project-1/Contact2.png" alt="background-img"/>
           </div>
           <div class="contact-text">
             <h1 class="Contactme">Contact Me</h1>
```

Comment:
- [ ] the following code for your Contactme class code, kindly remove `width: 801px;`  because it cause many extra white space in your design.
```css
.Contactme {
    position: absolute;
    width: 801px;
    height: 56px;
    font-style: normal;
    font-weight: 900;
    font-size: 40px;
    line-height: 52px;
    color: #fff;
}
```

Reply:
done